aries-site/src/examples/cardPreviews/textinput.js

```diff
@@ -4,14 +4,16 @@ import { TextInput, FormField } from 'grommet';
 export const TextInputPreview = () => {
   return (
     <FormField
-      style={{ borderRadius: '4px' }}
-      fill="horizontal"
-      border={{ color: 'focus', size: 'small' }}
+      style={{
+        boxShadow: '0 0 2px 2px #00E8CF',
+        borderRadius: '4px',
+      }}
```

Comment:
Think we need htmlFor="focus-id" on the FormField

Reply:
good catch!

Reply:
Plus wrap in Form?

Reply:
right! how did I fix all these and miss my own